Libraries/Components/TextInput/TextInput.js

```diff
@@ -703,6 +696,17 @@ export type Props = $ReadOnly<{|
    */
   multiline?: ?boolean,
 
+  /**
+   * Sets the number of lines for a `TextInput`. Use it with multiline set to
+   * `true` to be able to fill the lines.
+   */
+  numberOfLines?: ?number,
```

Comment:
We intend for this prop to be replaced by the `rows` prop (see above) as per https://github.com/necolas/discussions-and-proposals/blob/reduce-fragmentation/proposals/0000-reduce-fragmentation.md#textinput-component

Reply:
Make sense. Does it mean that I should change the name in this pr or are already working on one yourself?

Reply:
The `rows` prop (alias for `numberOfLines`) is already there on L530 ish. It might also make sense to rename (or alias) `maximumNumberOfLines` to something like `maxRows`.

Reply:
Is it android only for now? I haven't seen any logic related to rows on ios.

Reply:
I only see support for maximumNumberOfLines which is not the same. This pr adds support for numberOfLines so if rows hasn't been implemented yet for ios I can rename numberOfLines in the pr to rows.

Reply:
@necolas  are you interested in this pr? Do you want me to rename the prop to rows?

Reply:
Yes I already mentioned that. See https://github.com/facebook/react-native/pull/34488

Reply:
@necolas I've just renamed it. Sorry for the delay I was super busy with other tasks.